jsonschema2pojo-integration-tests/src/test/java/org/jsonschema2pojo/integration/json/JsonTypesIT.java

```diff
@@ -210,4 +210,29 @@ public void arrayAtRootProducesNoJavaTypes() throws Exception {
 
     }
 
+    @Test
+    @SuppressWarnings("rawtypes")
+    public void propertiesWithSameNameOnDifferentObjects() throws Exception {
```

Comment:
Would it make sense to add similar test also to `PlainYamlTypesIT` ?

Reply:
I think I will merge these two files into one parameterised test.